app/services/invoices/payments/retry_service.rb

```diff
@@ -17,7 +23,9 @@ def call
           return result.not_allowed_failure!(code: 'payment_processor_is_currently_handling_payment')
         end
 
-        SendWebhookJob.perform_later(webhook_type, invoice) if customer&.organization&.webhook_url?
+        if customer&.organization&.webhook_url?
```

Comment:
What do you think of extracting this block in a `deliver_webhook` method?

Reply:
extracted